src/citrine/resources/project.py

```diff
@@ -136,7 +142,7 @@ def predictor_evaluation_executions(self) -> PredictorEvaluationExecutionCollect
     @property
```

Comment:
Did we settle on this not needed a deprecation? (sorry if I missed the discussion)

Reply:
Correct. In the design workflows v2, branch is optional when listing. Without branch, it will list all design workflows in the project, just like today.

https://docs.corp.citrine.io/orion2/#operation/getApiV2Design-workflows